lib/modules/platform/gitlab/index.ts

```diff
@@ -912,9 +913,27 @@ export async function setBranchStatus({
     description,
     context,
   };
+
   if (targetUrl) {
     options.target_url = targetUrl;
   }
+
+  if (branchSha) {
+    const commitUrl = `projects/${config.repository}/repository/commits/${branchSha}`;
+    const LastPipelineId = z
```

Comment:
move to `schema.ts`

Reply:
Since it's not that big, maybe we just leave it here? IMO it will create more noise and cognitive load when extracted to separate file.

Reply:
it's not more noise because it's totally new code. so it doesn't matter where new code was added.

we also plan to move more interfaces to zod and we will forget about this inline schema.

it's also bad for performance to recreate the schema again and again on everything function call.

Reply:
@viceice I moved the zod schema to a new schema.ts file, can I ask you to check?